src/API/ApiEditSlot.php

```diff
@@ -44,11 +44,17 @@ public function execute() {
 			$params["slot"],
 			$params["summary"],
 			$params["append"],
-			$params["watchlist"]
+			$params["prepend"],
```

Comment:
I think we want watchlist before prepend if you have it in that order below.

Now that I think of it, when we do make it "Backwards compatible", shouldn't we set "suppress" as part of "watchlist"?

I mean, previously this:
watchlist=nochange
Would give what now is this:
suppress=true

Not the same at all.
So maybe it does make sense to break backwards compatibility on purpose? I am not sure.

Reply:
I don't consider fixing an issue that was clearly wrong (setting `watchlist` did not do *anything* with watchlists) a BC break.